aspnetcore/signalr/security.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Security Considerations in ASP.NET Core SignalR
+author: rachelappel
+description: Learn how to use authentication and authorization in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/29/2018
+uid: signalr/security
+---
+
+# Security Concepts in ASP.NET Core SignalR
+
+By [Andrew Stanton-Nurse](https://twitter.com/anurse)
+
+## Overview
+
+SignalR provides a number of security protections by default, but it is important to understand their limitations.
+
+### Cross-Origin Resource Sharing
+
+[Cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) can be used to allow cross-origin SignalR connections in the browser. If your JavaScript code is hosted on a different domain name from your SignalR app, you will have to enable the [ASP.NET Core CORS middleware](xref:security/cors) in order to allow the connection. In general, allow cross-origin requests only from domains you control. For example, if your site is hosted on `http://www.example.com` and your SignalR app is hosted on `http://signalr.example.com`, you should configure CORS in your SignalR app to only allow the origin `www.example.com`.
```

Comment:
- you will --> you
- Add " (CORS)" after the "Cross-origin resource sharing" link in the 1st sentence